src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html

```diff
@@ -18,11 +18,11 @@
 
 @this()
 
-@(params: Tabs)
+@(params: Tabs)(implicit messages: Messages)
 @import params._
 <div@id.mapNonEmpty {id => id="@id"} class="@toClasses("govuk-tabs", classes)"@toAttributes(attributes) data-module="govuk-tabs">
   <h2 class="govuk-tabs__title">
-    @HtmlFormat.escape(title)
+    @HtmlFormat.escape({title.getOrElse(messages("govuk.tabs.titleFallbackText"))})
```

Comment:
should we `@title.map(HtmlFormat.escape _).getOrElse(messages("govuk.tabs.titleFallbackText"))`

Reply:
not sure if the syntax for that is 100% correct

Reply:
@oscarduignan Suggested change made and pushed, thank. you.